functions/parquet/template.yaml

```diff
@@ -0,0 +1,36 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  parquet
+
+  Sample SAM Template for parquet
+  
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
+  CreateParquetTable:
```

Comment:
Love to see the SAM template. You can add the trigger here using an Event of type Schedule within SAM. If we want it to fire at something like midnight every given day of the week/month, a cron type would be most appropriate I think.

Also, you can add an IAM role here directly with something like the below

```yaml
  FunctionRole:
    Type: AWS::IAM::Role
    Properties:
      AssumeRolePolicyDocument:
        Version: 2012-10-17
        Statement:
          - Effect: Allow
            Principal:
              Service:
              - lambda.amazonaws.com
            Action:
              - 'sts:AssumeRole'
      ManagedPolicyArns:
        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
      Policies:
        - PolicyName: start-athena-query
          PolicyDocument:
            Version: '2012-10-17'
            Statement:
            - Effect: Allow
              Action:
              - athena:startQuery # TODO I am not sure what this actual action is
```

Reply:
Done? Except I couldn't figure out how to reference an environment variable inside the template. See line 25. I want to be able to reference `S3_BUCKET_LOCATION`, is that possible?

Reply:
If we were to do this all from scratch, we would actually create the S3 bucket in the template and then pass around it's name from there. However, since it's already created, I think you can use Parameters (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html) to do this and reference it in the Environment and Policy sections.